crates/modelardb_compression/src/lib.rs

```diff
@@ -952,19 +952,22 @@ pub mod test_util {
     /// will match `timestamps` and their structure will match [`StructureOfValues`]. If `Random` is
     /// selected, `min` and `max` is the range of values which can be generated. If `AlmostLinear`
     /// is selected, `min` and `max` is the maximum and minimum change that should be applied from
-    /// one value to the next. Returns the generated values as a [`Vec`].
+    /// one value to the next. Returns the generated values as a [`Vec<f32>`].
     pub fn generate_values(
         timestamps: &[i64],
         data_type: StructureOfValues,
         min: Option<f32>,
         max: Option<f32>,
     ) -> Vec<f32> {
+        let mut randomizer = thread_rng();
         match data_type {
             // Generates almost linear data.
             StructureOfValues::AlmostLinear => {
-                let a: i64 = thread_rng().gen_range(-10..10);
+                let mut a: i64 = 0;
+                while a == 0{
```

Comment:
Missing space between `0` and `{`, try reformatting the file with `rustfmt` and if that does not work add the space manually.

Reply:
Fixed.